wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -778,20 +797,21 @@ public boolean upLoadLandingZone(File uploadFile, DropZone dropZone)
             {
                 response.append(line);
             }
-            return true;
         }
         catch (MalformedURLException e)
         {
             // TODO Auto-generated catch block
             e.printStackTrace();
```

Comment:
This and the following catch still need to be updated (based on latest commit I see ( c2063cf)